src/test/java/com/google/sps/UserTest.java

```diff
@@ -39,6 +40,82 @@ public void userNotEquals() {
     User userA = new User("John");
     User userB = new User("Not John");
 
-    Assert.assertFalse(userA.equals(userB));
+    Assert.assertFalse((userA).equals(userB));
   }
+
+  @Test
+  public void setIDAndGetID(){
+      User userA = new User("Jeff");
+      long expected = 1112;
+      userA.setID(expected);
+      Long result = userA.getID();
+
+      Assert.assertEquals(Long.valueOf(expected), result);
```

Comment:
What's the Long.valueOf for on L53? Is it needed?

Reply:
Its so that it pulls the value of expected as a long and compares it to the result, its the syntax so that you can compare longs with assertEquals